tests/examples/test_flask_example.py

```diff
@@ -0,0 +1,16 @@
+from examples.flask_example import app, sinch_client
+
+
+def test_flask_create_app_get_endpoint(
+    auth_origin,
+    conversation_origin
+):
+    sinch_client.configuration.auth_origin = auth_origin
+    sinch_client.configuration.conversation_origin = conversation_origin
+    sinch_client.configuration.disable_https = True
+    app.testing = True
+    flask_client = app.test_client()
+
+    response = flask_client.post("/create_app")
+    assert response.status_code== 200
+    assert "sinch_app_id" in response.json
```

Comment:
What happens if `sinch_client.conversation.app.create()` fails? Will `sinch_app_id` contain an empty value? In this case, will this test succeed or fail?

Reply:
This is just a simple integration test - I want to know if examples are working correctly against current version of the SDK. When `sinch_client.conversation.app.create()` fails, flask returns HTTP 500, hence the test fails.